tools/rebuild_provision_request.rb

```diff
@@ -106,8 +106,10 @@ def parse_dialog_tabs
 
   def parse_source
     src_id = @provision_options.delete(:src_vm_id)
+    request_type = @provision_options.delete(:request_type)
     source = VmOrTemplate.find_by(:id => src_id)
     @output["template_fields"] = {'guid' => source.guid, 'name' => source.name}
+    @output["template_fields"].merge!('request_type' => request_type) if request_type
```

Comment:
@syncrou The `request_type` is also a column of the miq_request record.  If there is an easy way to reference that here we could include this information on the template_fields line and not need the conditional logic.

If not, I would add `.to_s`. to the request_type as it is the only value coming through as a symbol when the output is YAML.